examples/swig_native_python_boids_spatial3D/venv.bat

```diff
@@ -1,12 +0,0 @@
-@echo off
```

Comment:
I assume you didn't mean to actually just delete this file, rather you didnt force add the script in `python_boids_spatial3D_wrapped`, as `.bat` are gitignored.

Reply:
Do we really want a platform specific batch file? Would instructions not be better in the file header?

Reply:
I don't see the need, but @Robadob wanted it (as mentioned on slack). You have added one for the other python example.

Reply:
Batch file removed and readme added to each python example